src/main/java/org/assertj/core/api/AbstractPathAssert.java

```diff
@@ -0,0 +1,446 @@
+     *     $ ln -s bar foo
+     * </code></pre>
+     *
+     * <p>Given the above, the following assertion  will
+     * <strong>fail</strong>:</p>
+     *
+     * <pre><code class="java">
+     *     // Provided that "foo" is a Path for the above symbolic link
+     *     assertThat(foo).doesNotExist();
+     * </code></pre>
+     *
+     * @return self
+     *
+     * @throws AssertionError path exists
+     *
+     * @see Files#notExists(Path, LinkOption...)
+     * @see LinkOption#NOFOLLOW_LINKS
+     */
+    public S doesNotExist()
+    {
+        paths.assertNotExists(info, actual);
+        return myself;
+    }
+
+    /**
+     * Assert that a given path is absolute
+     *
+     * <p><strong>IMPORTANT NOTE:</strong> on {@link FileSystem}s with
+     * symbolic link support, the fact that a path is absolute <em>does not</em>
+     * guarantee that such a path is canonical. The only guarantee that this
+     * assertion makes is that the given path can be used on this filesystem
+     * to identify an object without having to rely on another path, whether the
+     * target of this path exists or not.</p>
+     *
+     * <p>Also note that the fact that a path is absolute does not mean that it
+     * is {@link Path#normalize() normalized}: {@code /foo/..} is absolute, for
+     * instance, but it is not normalized.</p>
+     *
+     * <p>Sample use:</p>
+     *
+     * <pre><code class="java">
+     *     assertThat(somePath).isAbsolute();
+     * </code></pre>
+     *
+     * <p>Sample paths on Unix and Windows which are absolute, or not:</p>
+     *
+     * <ul>
+     *     <li>{@code /foo/bar} on Unix: absolute;</li>
+     *     <li>{@code foo/bar} on Unix: not absolute;</li>
+     *     <li>{@code c:\foo} on Windows: absolute;</li>
+     *     <li>{@code foo\bar} on Windows: not absolute;</li>
+     *     <li>(surprise...) {@code c:foo} on Windows: <strong>not</strong>
+     *     absolute!</li>
+     *     <li>(surprise...) {@code \foo} on Windows: <strong>not</strong>
+     *     absolute!</li>
+     * </ul>
+     *
+     * @return self
+     *
+     * @throws AssertionError path is not an absolute path
+     *
+     * @see Path#isAbsolute()
+     * @see Path#toRealPath(LinkOption...)
+     */
+    public S isAbsolute()
+    {
+        paths.assertIsAbsolute(info, actual);
+        return myself;
+    }
+
+    /**
+     * Assert that a given {@link Path} is normalized
+     *
+     * <p>A path is normalized if it has no redundant components; typically, on
+     * both Unix and Windows, this means that the path has no "self" components
+     * ({@code .}) and that its only parent components ({@code ..}), if any, are
+     * at the beginning of the path.</p>
+     *
+     * Sample usage:
+     *
+     * <pre><code class="java">
+     *     assertThat(path).isNormalized();
+     * </code></pre>
+     *
+     * <p>For instance, {@code a/b} on Unix systems is normalized but {@code
```

Comment:
same comment as above, 
Move the various examples in the code section with a comment line before each case (not at the end).